src/TikaOnDotNet.Tests/text_extraction.cs

```diff
@@ -201,30 +206,52 @@ public void should_extract_msg()
         }
 
         [Test]
-        public void should_ocr_with_tesseract()
+        public void should_ocr_tif_with_tesseract()
         {
-            const string tesseractPath = @"c:\Program Files (x86)\Tesseract-OCR";
-
-            if (!File.Exists(Path.Combine(tesseractPath, "tesseract.exe")))
-                Assert.Ignore("Tesseract not found in the path '" + tesseractPath + "'");
+            if (!File.Exists(Path.Combine(TesseractPath, "tesseract.exe")))

```

Comment:
I will give it a try tomorrow because it's almost midnight over here in the Netherlands.
Never done something like this before but always nice to learn something new.